docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7' # specify docker-compose version
```

Comment:
Let us kindly move to H2 and remove the need to use Docker Compose.

Reply:
I think using Postgres as an alternative is better. Like the consideration of other application servers in your below comments, we could provide an option for the DB config in the production environment.

Reply:
The plan had been to use PostgreSQL on the cloud and indeed we can use Maven profiles and property replacement to do this down the line. If you look at the original (Spring) project and similar end-to-end examples, they all use embedded databases to keep the getting started experience very simple. Let us kindly follow the same pattern while introducing PostgreSQL a bit later.